changes/2822.bugfix.rst

```diff
@@ -0,0 +1 @@
+The Textual backend is now compatible with versions of Textual after v0.63.3.
```

Comment:
Should we pin the textual version? Or is that even worth doing when we don't have a automated suite to validate dependabot bumps?

Reply:
hmm...based on how dependencies are handled elsewhere, I'm leaning towards no. However, elsewhere we know when the latest versions of dependencies break. So, if we pin Textual, we know it won't randomly break....but then it'll also be harder to know when it does. So, given all this is still mostly POC, I think more information is better and leave the pin open-ended.

dependency survey
```
cocoa/pyproject.toml:64:dependencies = [
cocoa/pyproject.toml:65:    "fonttools >= 4.42.1, < 5.0.0",
cocoa/pyproject.toml:66:    "rubicon-objc >= 0.4.9, < 0.5.0",
cocoa/pyproject.toml:67:    "toga-core == {version}",
cocoa/pyproject.toml:68:]
--
gtk/pyproject.toml:66:dependencies = [
gtk/pyproject.toml:67:    "gbulb >= 0.5.3",
gtk/pyproject.toml:68:    "pycairo >= 1.17.0",
gtk/pyproject.toml:69:    "pygobject >= 3.46.0",
gtk/pyproject.toml:70:    "toga-core == {version}",
gtk/pyproject.toml:71:]
--
iOS/pyproject.toml:63:dependencies = [
iOS/pyproject.toml:64:    "fonttools >= 4.42.1, < 5.0.0",
iOS/pyproject.toml:65:    "rubicon-objc >= 0.4.9, < 0.5.0",
iOS/pyproject.toml:66:    "toga-core == {version}",
iOS/pyproject.toml:67:]
--
textual/pyproject.toml:67:dependencies = [
textual/pyproject.toml:68:    "textual >= 0.44.0",
textual/pyproject.toml:69:    "toga-core == {version}",
textual/pyproject.toml:70:]
--
winforms/pyproject.toml:64:dependencies = [
winforms/pyproject.toml:65:    "pythonnet >= 3.0.0",
winforms/pyproject.toml:66:    "toga-core == {version}",
winforms/pyproject.toml:67:]
```

Reply:
That's true - we pin our *development* dependencies, but we don't pin our runtime dependencies beyond the minimum + server major bump limits.